src/ocamlorg_frontend/layouts/layout.eml

```diff
@@ -73,6 +74,9 @@ inner =
       <% (match canonical with | Some canonical -> %>
       <link rel="canonical" href="https://ocaml.org<%s canonical %>">
       <% | None -> ()); %>
+      <% (match alternate with | Some alternate -> %>
+      <link rel="alternate" href="<%s alternate %>" title="The OCaml Planet" type="application/atom+xml">
+      <% | None -> ()); %>
```

Comment:
Seems to be the wrong place to assume that alternate always is a link to the OCaml Planet.

I would rather add an ?(extra_head_html = "") parameter on the main layout where we can add all kinds of varying content into the `<head>` section of the page. Either that, or make alternate a record `{href : string; title: string; type_ : string}`. However, I think this is a one-off situation to place a `<link rel="alternate" ...>`, so `extra_head_html` seems less complex and more flexible.